README.md

```diff
@@ -117,6 +117,7 @@ The following suites are available to run:
 * deploy
 * content
 * contrib
+* contrib-dev
```

Comment:
Do these suites have documentation in the README, or is it presumed that they are self-documenting in the yml files?